src/Main.java

```diff
@@ -1,5 +1,95 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //task 1
+        System.out.println("task 1");
+        char clientOS = 0;
+        switch (clientOS) {
+            case 0:
+                System.out.println("Установите версию приложения для iOS по ссылке");
+                break;
+            case 1:
+                System.out.println("Установите версию приложения для Android по ссылке");
+                break;
+            default:
+                System.out.println("Извините, такого приложения у нас нет");
+        }
+
+        //task 2
+        System.out.println("task 2");
+        char clientOS2 = 2;
```

Comment:
зачем заводить новую переменную? почему бы просто не обнулить уже объявленную?
```suggestion
        clientOS = 2;
```